src/System.Private.ServiceModel/src/System/ServiceModel/Channels/StreamedFramingRequestChannel.cs

```diff
@@ -175,14 +180,19 @@ protected override void OnClosed()
             ChannelBindingUtility.Dispose(ref _channelBindingToken);
         }
 
+        protected internal override Task OnCloseAsync(TimeSpan timeout)
+        {
+            return base.WaitForPendingRequestsAsync(timeout);
+        }
+
         protected override IAsyncResult OnBeginClose(TimeSpan timeout, AsyncCallback callback, object state)
         {
-            return base.BeginWaitForPendingRequests(timeout, callback, state);
+            return OnCloseAsync(timeout).ToApm(callback, state);
         }
 
         protected override void OnEndClose(IAsyncResult result)
         {
-            base.EndWaitForPendingRequests(result);
```

Comment:
As you are removing the call to the base APM method and instead calling in to the Async version, the base APM method(s) should be removed.